recipes/sortmerna/4.2.0/build.sh

```diff
@@ -0,0 +1,9 @@
+#
+# FILE: build.sh
+# Created: Dec 08, 2019 Sun
+#
+# standard Conda build script for building sortmerna v4.2.0
+# Uses pre-built binaries from Sortmerna GitHub Releases
+#
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+mv ./bin/sortmerna $PREFIX/bin
```

Comment:
In general, bioconda prefers building packages from source unless there are specific reasons why that isn’t possible.

Reply:
Sortmerna versions >=4.x.x require gcc >=9.2.1 and RocksDB library version not available on the stock Debian distros i.e. has to be built. This gets rather frustrating with conda-build. For the time being I decided to go with the pre-built binaries. I'm working on it though, and the next release hopefully will work with sources. I know that for example [deepvariant](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/deepvariant/meta.yaml) also uses pre-builts.

Reply:
Sounds reasonable.